### PR TITLE
Add data-model.md to explian objectes in API

### PR DESCRIPTION
**Pull request by hanahmily**
Signed-off-by: Gao Hongtao <hanahmily@gmail.com>

### CODE REVIEW

File: docs/concept/data-model.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+`Measure` supports the types of the following fields:
+
+* **ZSTD** : Zstandard is a real-time compression algorithm, that provides high compression ratios. It offers a very wide range of compression/speed trade-offs, while being backed by a very fast decoder. For BanyanDB focus on speed.
+
+Another option named `interval` plays a critical role in encoding. It indicates the time range between two adjacent data points in a time series and implies that all data points belonging to the same time series are distributed based on a fixed interval. A better practice for the naming measure is to append the interval literal to the tail, for example, `service_cpm_minute`. It's a parameter of `GORILLA` encoding method.
+
+[Measure Registration Operations](../api-reference.md#measureregistryservice)
+
+#### TopNAggregation
```
Comment (hanahmily) [inline docs/concept/data-model.md:85]: @lujiajing1126 Would you please check this section?

**Comment by wu-sheng** [comment]
The images are better hosted in here https://github.com/apache/skywalking-website/tree/master/static

Once they are merged, you could use a static link for it.
Notice, the folder should be versionalized in case some are changed from version to version.

File: docs/concept/data-model.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+counters_number: 10000
+lru_size: 10
+```
+
+`endpoint_cpm_minute_top_bottom` is watching the data ingesting of `endpoint_cpm_minute` to generate top 1000 and bottom 1000 entity cardinalities respectively. If you want to only calculate Top 1000 or Bottom 1000, the `field_value_sort` could be `DESC` or `ASC`.
+
+* SORT_DESC: Top-N. In a series of `1,2,3...1000`. Top10's result is `1000,999...991`.
+* SORT_ASC: Bottom-N. In a series of `1,2,3...1000`. Bottom10's result is `1,2...10`.
+
+Tags in `group_by_tag_names` need to be as dimensions. The query phase could retrieve these tags. Tags no in `group_by_tag_names` could be dropped in the calculating phase.
```
Comment (lujiajing1126) [inline docs/concept/data-model.md:114]:
```suggestion
Tags in `group_by_tag_names` are used as dimensions. These tags can be searched (only equality is supported) in the query phase. Tags do not exist in `group_by_tag_names` will be dropped in the pre-calculating phase.
```

File: docs/concept/data-model.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+`counters_number` denotes the number of entity cardinality. As the example shows, calculating the Top 100 among 10 thousand is easier than among 10 million.
```
Comment (lujiajing1126) [inline docs/concept/data-model.md:116]:
```suggestion
`counters_number` denotes the number of entity cardinality. As the above example shows, calculating the Top 100 among 10 thousands is easier than among 10 millions.
```

**Review by lujiajing1126 — CHANGES_REQUESTED** [review]
Some suggestions for improvement.

And there are typos in `docs/assets/index-binding.png`

**Comment by lujiajing1126** [comment]
image

`Sream` should be `Stream`?

**Review by lujiajing1126 — APPROVED** [review]
LGTM

File: docs/concept/data-model.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+* data model
+* data retrieval
+
+You can also find [examples](../crud/) of how to interact with BanyanDB using [bydbctl](../clients.md#command-line), how to create and drop groups, or how to create, read, update and drop streams/measures.
+
+## Structure of BanyanDB
+
+The hierarchy that data is organized into **streams**, **measures** and **properties** in groups.
+
+![Structure of BanyanDB](https://skywalking.apache.org/doc-graph/banyandb/v0.2.0/structure.png)
```
Comment (wu-sheng) [inline docs/concept/data-model.md:15]: This is how we refer static files hosted on the website.

**Pull request merged**